pkg/controller/support/credentials/cloudauth/acr/acr.go

```diff
@@ -116,9 +118,11 @@ func (a *acrProvider) authenticate(ctx context.Context, server string) (*types.A
 		armAccessToken,
 	)
 	if err != nil {
+		logger.Error(err, "Token refresh failed.")
 		return nil, fmt.Errorf("failed to generate ACR refresh token: %w", err)
 	}
 
+	logger.Info("Successfully authenticated with ACR")
```

Comment:
I wasn't sure if these "success logs" when authenticating with a certain cloud provider were ok, or if that might be a security issue.

Reply:
This is totally fine. These logs show up in the build logs that are already governed by RBAC, right?